internal/auth0/client/resource.go

```diff
@@ -283,11 +284,12 @@ func NewResource() *schema.Resource {
 							},
 						},
 						"ios": {
-							Type:        schema.TypeList,
-							Optional:    true,
-							Computed:    true,
-							MaxItems:    1,
-							Description: "Configuration settings for i0S native apps.",
+							Type:         schema.TypeList,
+							Optional:     true,
+							Computed:     true,
+							MaxItems:     1,
+							Description:  "Configuration settings for i0S native apps.",
+							AtLeastOneOf: []string{"mobile.0.android"},
```

Comment:
Should this be `"mobile.0.ios"`?

Reply:
No, when using AtLeastOneOf the current resource attribute is already included.